src/stores/models/Transaction.js

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+
+import { Token, TransactionType } from '../../constants';
+import { gasToQtum, satoshiToDecimal } from '../../helpers/utility';
+
+export default class Transaction {
+  type = ''
+  txid = ''
+  status = ''
+  createdTime = ''
+  blockNum = 0
+  blockTime = ''
+  gasLimit = 0
+  gasPrice = 0
+  gasUsed = 0
+  version = 0
+  senderAddress = ''
+  receiverAddress = ''
+  topicAddress = ''
+  oracleAddress = ''
+  name = ''
+  optionIdx = 0
+  token = ''
+  amount = ''
+  topic
+
+  constructor(transaction, app) {
+    Object.assign(this, transaction);
+    this.app = app;
```

Comment:
so we're not actually passing the `app` store down. We should either pass it in, or remove this prop for now until we need it :)

Reply:
fixed, accidentally left that in there :P